config/locales/en.yml

```diff
@@ -38,4 +41,14 @@ en:
       error: 'You have not updated company successfully!'
     destroy:
       success: 'You have deleted company successfully!'
-      error: 'You could not able to destroy company!'
\ No newline at end of file
+      error: 'You could not able to destroy company!'
+  policies:
+    create:
+      success: 'You have created policy successfully!'
+      error: 'You have not created policy successfully!'
```

Comment:
@GauravMakwana-BTC : Display the relevant error messages., Update this at other places